examples/config_tiny_llama.py

```diff
@@ -97,7 +97,6 @@
     logging=LoggingArgs(),
     tokens=tokens,
     data=DataArgs(dataset=dataset, seed=seed),
-    profiler=None,
```

Comment:
Set `Optional[ProfilerArgs] = None` instantiation in [`nanotron/config/config.py#L334`](https://github.com/huggingface/nanotron/blob/main/src/nanotron/config/config.py#L334)

```python
profiler: Optional[ProfilerArgs] = None
```